decentralization.adoc

```diff
@@ -270,10 +270,10 @@ https://www.reddit.com/r/Bitcoin/comments/s82t2n/comment/htdte7w/?utm_source=sha
 the Bitcoin subreddit], compares this to the English language in an
 intriguing way:
 
-[quote, Gregory Maxwell, r/Bitcoin subreddit]
+[quote, Gregory Maxwell, r/Bitcoin subreddit (2022)]
 ____
 Many people have a hard time understanding autonomous systems, there
-are many in their lives things like the english language-- but people
+are many in their lives, things like the english language-- but people
```

Comment:
Quote. Leave as-is.

Reply:
I know, but I found it hard to understand and I automatically added the comma. However, you're right, I'll leave it as it is